src/diffusers/utils/import_utils.py

```diff
@@ -186,15 +211,7 @@ def _is_package_available(pkg_name: str):
 _gguf_available, _gguf_version = _is_package_available("gguf")
 _torchao_available, _torchao_version = _is_package_available("torchao")
 _bitsandbytes_available, _bitsandbytes_version = _is_package_available("bitsandbytes")
-_torchao_available, _torchao_version = _is_package_available("torchao")
```

Comment:
is this intended?

Reply:
It's duplicate

https://github.com/huggingface/diffusers/blob/68bb9553d3f774a2f539f6eedd0221e9f5bdb3ce/src/diffusers/utils/import_utils.py#L212